src/current/v23.2/known-limitations.md

```diff
@@ -165,7 +138,17 @@ In cases where the partition definition includes a comparison with `NULL` and a
 
 {% include {{page.version.version}}/known-limitations/sql-cursors.md %}
 
-### `SELECT FOR UPDATE` locks are dropped on lease transfers  and range splits/merges
+### Limitations for `SELECT FOR UPDATE`
+
+- [`SELECT FOR UPDATE`]({% link {{ page.version.version }}/select-for-update.md %}) places locks on each key scanned by the base index scan. This means that even if some of those keys are later filtered out by a predicate which could not be pushed into the scan, they will still be locked.
+
+    [Tracking GitHub issue](https://github.com/cockroachdb/cockroach/issues/75457)
+
+- [`SELECT FOR UPDATE`]({% link {{ page.version.version }}/select-for-update.md %}) only places an unreplicated lock on the index being scanned by the query. This diverges from PostgreSQL, which aquires a lock on all indexes.
+
+    [Tracking GitHub issue](https://github.com/cockroachdb/cockroach/issues/57031)
+
+### `SELECT FOR UPDATE` locks are dropped on lease transfers and range splits/merges
```

Comment:
This is another one worth qualifying. Maybe we should say

> Under serializable isolation, SELECT FOR UPDATE locks are dropped on lease transfers and range splits/merges. This limitation does not exist when using SELECT FOR UPDATE under read committed isolation or with `enable_durable_locking_for_serializable` set to true under serializable isolation.

Reply:
This is already fixed on the published version. @mikeCRL, however, I removed it from this list and kept it to the SELECT FOR UPDATE doc, because that somehow made sense at the time. Now I feel it's better off being on this doc as well, so if there is a merge conflict when rebasing this PR, I think we should simply keep this heading + include intact so that it's brought back. Sorry for the extra complication.